### PR TITLE
Implement Vue Realtime tracker application

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: src/components/Application.vue [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+      });
+    });
+
+
+
+    this.polling = setInterval(() => {
+      this.onlineUsers = membersData
+      this.markers = membersData.map((mem) => {
+        if (JSON.stringify(this.userlocation) == JSON.stringify(mem.data)) {
+          return {...mem.data, icon: 'http://maps.google.com/mapfiles/ms/icons/blue-dot.png'}
```
Comment (Srushtika) [inline src/components/Application.vue:88]: Syntax Error: Unexpected token (88:18) here. This error doesn't let me test the app generally, please fix it.

Comment (ogwurujohnson) [reply src/components/Application.vue:88]: i don't understand the error, you are getting

Comment (ogwurujohnson) [reply src/components/Application.vue:88]: let me look at it again

Comment (ogwurujohnson) [reply src/components/Application.vue:88]: fixed, you can have a look now

**Pull request closed without merging**